vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -243,7 +243,8 @@ public void setValue(LocalDateTime value) {
         // is fixed: https://github.com/vaadin/vaadin-time-picker-flow/issues/77
         getElement().executeJs("return this.value;")
                 .then(jsonValue -> super.setValue(
-                        LocalDateTime.parse(jsonValue.asString())));
+                        jsonValue.asString().isEmpty() ? null :
+                                LocalDateTime.parse(jsonValue.asString())));
```

Comment:
I think this whole `executeJs()` block could be skipped if setting value to null, but I guess this works too.

Reply:
Ok. I didn't check what the actual issue is that this workaround is for, so don't know if it's relevant for null value.

Reply:
This is about getting the "adjusted value" from client side after setting a value since depending on how accurate value you set (with milliseconds or not) and the settings (step etc.) the client side might drop extra digits from the end of the value.

When you clear the value (e.g. setting to null) there should be no such adjustment/change to the value on client side, so we shouldn't need to do this round trip.